gems/aws-sdk-core/spec/aws/log/formatter_spec.rb

```diff
@@ -43,15 +47,18 @@ def format(pattern, options = {})
             config: {
               nested: true,
               path: Pathname.new(__FILE__),
+              tmpfile:,
```

Comment:
This will fail older rubies. I would also just call this "file" to match the case statement.

Reply:
Fixed, although the `file` key is marked as sensitive [here](https://github.com/aws/aws-sdk-ruby/blob/version-3/gems/aws-sdk-core/lib/aws-sdk-core/log/param_filter.rb#L19), so it would be filtered in the output. I've left it as `tmpfile`, but can change that to something else if you wish.